### PR TITLE
added the duplicate enrollment trigger as well as its helper method a…

### PR DESCRIPTION
**Pull request by mikeHartzell95**
…nd tets.

### CODE REVIEW

File: force-app/main/default/classes/DuplicateEnrollmentException.cls [hunk 1]
```diff
@@ -0,0 +1,2 @@
+public class DuplicateEnrollmentException extends Exception {
```
Comment (mruizrevature) [inline force-app/main/default/classes/DuplicateEnrollmentException.cls:1]: Please don't include these empty files inside of your commits.

File: force-app/main/default/classes/DuplicateEnrollmentTriggerTest.cls [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        Class__c c = [SELECT Id FROM Class__c LIMIT 1];
+        ClassEnrollment__c ce = new ClassEnrollment__c();
+        ce.Student__c = s.Id;
+        ce.Class__c = c.Id;
+        ce.Credits__c = 1;
+        insert ce;
+    }  
+
+    @isTest
+    public static void testSingleInsert()
```
Comment (mruizrevature) [inline force-app/main/default/classes/DuplicateEnrollmentTriggerTest.cls]: Your test names should include the methods in which you are testing. It's hard to know what this is for without context.

File: force-app/main/default/classes/duplicateEnrollmentTriggerHelper.cls [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    public static final String DUPLICATE_ENROLLMENT = 'That student is already enrolled in the class.';
+
+    
+    public static void preventDuplicateEnrollment(List<ClassEnrollment__c> enrollmentsBeingInserted)
+    {
+        List<ClassEnrollment__c> enrollments = [SELECT Student__c, Class__c FROM ClassEnrollment__c];
+        List<ClassEnrollment__c> deniedEnrollments = new List<ClassEnrollment__c>();
+
+        for(ClassEnrollment__C existingEnrollment: enrollments)
+        {
```
Comment (mruizrevature) [inline force-app/main/default/classes/duplicateEnrollmentTriggerHelper.cls:16]: Add some comments for context.

**Review by mruizrevature — APPROVED** [review]
Not bad but it takes a while to figure out what's going on. Adding comments will help that. Also, it's very common add comments to your methods.

File: force-app/main/default/classes/DuplicateEnrollmentException.cls (hunk 1 above)
Comment (mikeHartzell95) [reply force-app/main/default/classes/DuplicateEnrollmentException.cls:1]: This is my custom exception. I thought we only needed to create a class that extended exception for that. Is there something else that is supposed to be in the body?

**Pull request merged**